src/main/pg/rx.h

```diff
@@ -50,6 +53,7 @@ typedef struct rxConfig_s {
     uint8_t max_aux_channel;
     uint8_t rssi_src_frame_errors;          // true to use frame drop flags in the rx protocol
     int8_t rssi_offset;                     // offset applied to the RSSI value before it is returned
+    uint8_t rssi_sbus_lpf_period;           // Period of the cutoff frequency for the SBus frame RSSI filter (in 0.1 s)
```

Comment:
Please read https://github.com/betaflight/betaflight/blob/master/docs/development/ParameterGroups.md and fix accordingly.

Reply:
@mikeller What's the concern here? To address each rule in turn...
1. rssi_sbus_lpf_period is placed in the RSSI group, so I think we're good here.
2. This is independent of configuration, so that's OK
3. NA
4. Inclusion here is on a par with rssi_src_frame_errors, so I don't see any problem
5. It's at the end of the group, so OK there.
**6. I think you got me here... I added a new default value of 30 in the PG_DECLARE, so I should have updated the version of the parameter group.** Easily missed, this one. I've updated the version in PG_REGISTER_WITH_RESET_FN of rxConfig to 3.
7. NA
8. NA
9. NA

Was it just point 6?

Reply:
@SteveCEvans: The concern is that the rules for parameter groups should be understood and applied when handling parameter groups, or all kinds of breakages will occur.

> 5.  New parameters should always be appended at the end of the parameter group.

Reply:
Ah, understood. I'd interpreted this to mean the end of the logical group of definitions in settings.c and this was the last rssi_* variables, but I now appreciate it meant the end of the structure, not any logical grouping.